src/WebJobs.Extensions.DurableTask/IPlatformInformation.cs

```diff
@@ -83,6 +83,12 @@ public interface IPlatformInformation
         /// <returns> True if the plan is Consumption. Otherwise, False.</returns>
         bool IsInConsumptionPlan();
 
+        /// <summary>
+        /// Determine the underlying plan is Consumption or not.
+        /// </summary>
+        /// <returns> True if the plan is Consumption. Otherwise, False.</returns>
+        bool IsManagedAppEnvironment();
```

Comment:
I suggest we call this `IsManagedContainerAppsEnvironment()` since this is specific to ACA + Functions. Just calling it "managed" is a bit too ambiguous.

Also, the copy/paste comments need to be updated.